codespell_lib/data/dictionary.txt

```diff
@@ -39115,6 +39115,7 @@ mountin->mounting, mountain, mountie, mount in,
 mountpiont->mountpoint
 mountpionts->mountpoints
 mouspointer->mousepointer
+mouting->mounting
```

Comment:
It looks like [mouting](https://www.oed.com/search/dictionary/?q=mouting) is a variant of moulting | molting. Should we move it to the _rare_ dictionary?